variables.tf

```diff
@@ -163,6 +163,16 @@ variable "enable_s3_endpoint" {
   default     = false
 }
 
+variable "enable_ssm_endpoint" {
+  description = "Should be true if you want to provision an SSM interface endpoint to the VPC"
+  default     = false
+}
+
+variable "ssm_endpoint_security_group_ids" {
+  description = "List of security group IDs applied to the SSM interface endpoint."
+  type        = "list"
```

Comment:
added the default setting.

Any idea how to deal with the requirement that subnets have to be unique within AZs? It's not possible just to add all public/private subnets as this could add multiple subnets of the same AZ(s).